devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -102,6 +130,11 @@ public abstract class Ssd13xx : IDisposable
         /// </summary>
         protected I2cDevice _i2cDevice { get; set; }
 
+        /// <summary>
+        /// Gets or sets Screen rotation.  0 = no rotation, 1 = 90, 2 = 180, 3 = 270.
+        /// </summary>
+        public int Rotation { get; set; } = 0;
```

Comment:
This is a good idea.  Since it's actually a reference to a rotation rather than an absolute orientation, I changed this to RotationOptions.  If the display is 64x128 then it defaults into portrait with no rotation but if the display is 128x64 than it defaults into landscape with no rotation.  Also these little screens don't necessarily have an up or down so it could be confusing to have a Portrait vs Portrait180 just depending on how you're holding the LCD screen.  I think just having everything with reference to the default orientation is more straightforward.